docs/local_development_kube.md

```diff
@@ -10,116 +10,48 @@ It's like having a conversation with your cluster.
 
 This guide assumes you have the following tools installed:
 
+- [brew](https://brew.sh/) if you are on linux
 - [Docker](https://docs.docker.com/get-docker/)
 - [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 - [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/)
-  * [mkcert](https://github.com/FiloSottile/mkcert) 
+  * [mkcert](https://github.com/FiloSottile/mkcert)
 - [ctlptl](https://github.com/tilt-dev/ctlptl)
 - [Tilt](https://docs.tilt.dev/install.html)
   * [helm](https://helm.sh/docs/intro/install/)
   * [helmfile](https://github.com/helmfile/helmfile)
   * [secrets](https://github.com/jkroepke/helm-secrets/wiki/Installation)
-  * [sops](https://github.com/getsops/sops)
-
-
-### SOPS configuration
-
-**Generate a SOPS key**
-
-For this specific step you need to have the `age-keygen` tool installed.
-See https://github.com/FiloSottile/age.
-Then generate a key:
-
-```bash
-age-keygen -o my-age.key
-```
-
-**Install the SOPS key**
-
-Read the SOPS documentation on how to install the key in your environment.
-https://github.com/getsops/sops?tab=readme-ov-file#22encrypting-using-age
-
-On Ubuntu it's like:
-
-```bash
-mkdir -p ~/.config/sops/age/
-cp my-age.key ~/.config/sops/age/keys.txt
-chmod 400 ~/.config/sops/age/keys.txt
-```
-
-**Add the SOPS key to the repository**
-
-Update the [.sops.yaml](../.sops.yaml) file with the **public** key id you generated.
-
-
-### Helmfile in Docker
-
-If you use helmfile in Docker, you may need an additional configuration to make 
-it work with you age key.
-
-You need to mount `-v "${HOME}/.config/sops/age/:/helm/.config/sops/age/"`
-
-```bash
-#!/bin/sh
-
-docker run --rm --net=host \
-   -v "${HOME}/.kube:/root/.kube" \
-   -v "${HOME}/.config/helm:/root/.config/helm" \
-   -v "${HOME}/.config/sops/age/:/helm/.config/sops/age/" \
-   -v "${HOME}/.minikube:/${HOME}/.minikube" \
-   -v "${PWD}:/wd" \
-   -e KUBECONFIG=/root/.kube/config \
-   --workdir /wd ghcr.io/helmfile/helmfile:v0.150.0 helmfile "$@"
-```
-
 
 ## Getting started
 
 ### Create the kubernetes cluster
 
 Run the following command to create a kubernetes cluster using kind:
 
-```bash
-./bin/start-kind.sh
-```
-
-**or** run the equivalent using the makefile
-
 ```bash
 make start-kind
 ```
 
-### [Optional] Install the secret
+### Install the secret
 
-You don't need to do this if you are running the stack with keycloak.
+You might not need to do this if you are run the stack with keycloak. 
 
 ```bash
 make install-external-secrets
 ```
 
 ### Deploy the application
 
-```bash
-# Pro Connect environment
-tilt up -f ./bin/Tiltfile 
```

Comment:
This is the command I use, why remove it? It's common for Tilt users to use it like this

Reply:
but since you made the `make tilt-up` command, I thought it would lighten the doc to not put both commands everytime. But I lack experience wiht tilt. If this command is especially familiar with the community, maybe we should keep this command and add `make tilt-up` in parenthesis or something like that.